server/coordinator.go

```diff
@@ -452,7 +494,7 @@ func (c *coordinator) runScheduler(s *scheduleController) {
 		select {
 		case <-timer.C:
 			timer.Reset(s.GetInterval())
-			if !s.AllowSchedule() {
+			if !s.AllowSchedule() && !s.isPaused() {
```

Comment:
I think we can invoke `isPaused` inside `AllowSchedule` which makes the API clearer.

Reply:
Sorry to have this mistake..., my unit test just use `isPaused` to jugde the whether the scheduler is paused, so it dosen't test out this error.